prem/VennDiagram.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+    <html>
+    <head>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>
+            VennDiagram
+        </title>
+        <style>
+            .main-section {
+            position: absolute;
+            width: 30%;
+            height: 30%;
+            margin: auto;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+        }
+        .green-circle,.orange-circle,.pink-circle{
+            width: 50%;
+            height:75%;
+            border-radius: 50%;
+            position: absolute;
+        }
+        .green-circle{
+            background-color:#00fb00;
+            display: inline-block;
+            margin-left:5%;
+
+        }
+        .orange-circle{
+            background-color: #f1a300;
+            display: block;
+            margin-left:42%; 
+        }
+        .pink-circle{
+            background-color: rgb(240, 158, 240);
+            display: block;

```

Comment:
Why you have added "display:block" ?

Reply:
So, there are three circles in the Venn diagram, two circles should be in same line (inline) while the third circle is in next block. So ,for tat purpose, display:block is mentioned for the third circle.